### PR TITLE
Feature: Report

### PR DESCRIPTION
**Pull request by qsqus**
# 🏴‍☠️ `Feature/Report` Branch to `Main` Branch Pull Request 🏴‍☠️

## Changes Made:

1. Added an emergency button savvy enough to be used once by every scallywag!
2. Introduced a report feature, arrr! Now ye can snitch on the mischievous crewmates.
3. Implemented a report screen that displays the sorry soul ye've accused!
4. Fixed issues plaguin' the use of cameras and vents. No more ghostly problems!
5. Added spawn positions and meeting positions to make the ship feel more like home.
6. Unveiled a shiny sabotage timer display – keep an eye on the countdown, me hearties!
7. Installed a kill cooldown at the start of every round. A pirate's gotta have some patience!
8. Adjusted the voting screen for a smoother decision-makin' process.
9. Fixed some pesky issues that were lurkin' in the shadows.
10. Added new lobby settings – now ye can adjust the emergency meeting cooldown to yer likin'!

## Yo-ho-ho and a bottle of merge! 🪣🏴‍☠️

This be a mighty fine set of changes, ready to set sail into the main branch. Feel free to inspect the code and give yer hearty approval. If there be any concerns, speak up, and we'll navigate through the treacherous waters together.

May the code be ever in yer favor, Captain Main Branch! ⚓🌊

### CODE REVIEW

File: globals/game_manager/game_manager.gd [hunk 1]
```diff
@@ -299,7 +306,10 @@ func reset_game():
 func new_round():
 	# Resetuje system głosowania.
 	_reset_votes()
-
+	
+	next_round_started.emit()
+	set_input_status(true)
```
Comment (thedavesky) [inline globals/game_manager/game_manager.gd:311]: Sądzę, że ustawianie input'u powinno być gdzieś indziej, ale naprawię to w nowym feature.

File: scenes/maps/main_map/main_map.gd [hunk 2]
```diff
@@ -58,8 +63,9 @@ func _spawn_player(id: int):
 
 	# Ustawia startową pozycję gracza.
 	if multiplayer.is_server():
-		player.position = Vector2(randi_range(0, 100), randi_range(0, 100))
-	
+		player.position = start_positions[position_idx].position
```
Comment (thedavesky) [inline scenes/maps/main_map/main_map.gd:66]: Nie trzeba liczyć pozycji, wystarczy jak w lobby:
```
GameManager.get_registered_players().keys().find(id)
```

File: scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png
Comment (thedavesky) [inline scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png:1]: Nazwa pliku nie powinna rozpoczynać się od underscore'a.

File: scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png.import
Comment (thedavesky) [inline scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png.import:1]: Jak w PNG.

File: scenes/maps/main_map/scenes/report/emergency_button/emeC613.tmp
Comment (thedavesky) [inline scenes/maps/main_map/scenes/report/emergency_button/emeC613.tmp:1]: Nie dodajemy plików *.tmp.

File: scenes/maps/main_map/scenes/report/emergency_button/emeCA4.tmp
Comment (thedavesky) [inline scenes/maps/main_map/scenes/report/emergency_button/emeCA4.tmp:1]: Nie dodajemy plików *.tmp.

File: globals/game_manager/game_manager.gd (hunk 1 above)
Comment (qsqus) [reply globals/game_manager/game_manager.gd]: poprawiłem, jest teraz w emergency_button

File: scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png
Comment (qsqus) [reply scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png:1]: poprawione, przy okazji poprawiłem _czat na czat

File: scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png.import
Comment (qsqus) [reply scenes/maps/main_map/scenes/report/emergency_button/assets/_emergency_meeting.png.import:1]: poprawione

File: scenes/maps/main_map/scenes/report/emergency_button/emeC613.tmp
Comment (qsqus) [reply scenes/maps/main_map/scenes/report/emergency_button/emeC613.tmp:1]: nie wiem co to jest - usunąłęm

File: scenes/maps/main_map/scenes/report/emergency_button/emeCA4.tmp
Comment (qsqus) [reply scenes/maps/main_map/scenes/report/emergency_button/emeCA4.tmp:1]: nie wiem co to jest - usunąłęm

**Pull request merged**